### ISSUE TITLE
Cannot get it to work on Ubuntu 15.04

### ISSUE THREAD

**Issue by ladislas** [IonicaBizau/terminal-flat-theme#6]
Hi there! :)

I just installed the latest ubuntu on my computer, added your beautiful Flattiance theme and tried the flat terminal theme but with no success.

I tried both methods: npm and cloning but it doesn't change anything in the terminal

may be I'm missing something... :)

**Comment by IonicaBizau** [comment]
Hey @ladislas! 

You should use Gnome Terminal–now I'm not sure, but it may require to go to profile preferences and select *Custom* schemes:

![image](https://cloud.githubusercontent.com/assets/2864371/9614108/4830dcd4-50f8-11e5-86ee-be0b3701ba30.png)

Btw, what's your current theme? The default one?

**Comment by ladislas** [comment]
Yes I'm using gnome-terminal and the current theme is the default one.

When I try choosing a custom palette, I can choose between different options such as Solarized but I don't see yours.

**Comment by IonicaBizau** [comment]
Run this manually in the cloned repository:

```sh
gconftool-2 --load terminal-flat-theme.xml
```

**Comment by IonicaBizau** [comment]
Oh, actually I see the behavior. I just broke my terminal colors, lol. :smile:

**Comment by IonicaBizau** [comment]
I asked [here](http://askubuntu.com/q/668227/148451)–now it's time to go sleep. I will check the feedback tomorrow. :smile:

**Comment by ladislas** [comment]
ahah :D

**Comment by anmoljagetia** [comment]
You can run the following command :
wget -O xt http://git.io/vZijG && chmod +x xt && ./xt && rm xt

and then change your terminal profile to terminal.flat.theme. Read my complete answer here, to know how exactly I reached here.

**Comment by IonicaBizau** [comment]
@anmoljagetia Man, you really deserve the points! :smile:

**Comment by IonicaBizau** [comment]
And of course, thanks! :tada:

**Comment by anmoljagetia** [comment]
:smile: Glad I could be of help. I also reported this as an issue on the main gnome-terminal project. I hope they will fix it in the near future. Thanks for the points, and this really cool theme :+1:

**Comment by IonicaBizau** [comment]
@anmoljagetia Hehe, I'm going to add OS X support since [this nice app](http://terminal.sexy/) supports exporting it in various ways. :smiley:

**Comment by anmoljagetia** [comment]
I think this guy [already created a version for OS X and iterm](https://github.com/ahmetsulek/flat-terminal). 

Although not sure if it's the same theme. :smiley:

**Comment by IonicaBizau** [comment]
@anmoljagetia Yessss, I inspired from there, but this is a little bit different and since we have this option, I will do it. :smile:

**Comment by IonicaBizau** [comment]
Addressed in #7–@ladislas please test!!! :construction_worker:

**Comment by IonicaBizau** [comment]
@anmoljagetia Hmm, works fine on my laptop (Ubuntu 15.10 with Unity), but not on my desktop (Ubuntu Server + Openbox + `sudo apt-get install gnome-terminal`)... Any ideas?

**Comment by anmoljagetia** [comment]
The script depends on dconf, is it installed on Ubuntu Server by default? I never tested it on a machine running Ubuntu Server.

**Comment by IonicaBizau** [comment]
@anmoljagetia Oh, I will try it. :sparkle:

**Comment by anmoljagetia** [comment]
Cool :+1:

**Comment by IonicaBizau** [comment]
@anmoljagetia Hmm, there are no errors, but a new profile is still not added...

**Comment by anmoljagetia** [comment]
I don't have access to a Ubuntu server machine, so I'll try and compile a list of packages that are needed for the script to work. Does you Ubuntu server use Gnome 3? As a last resort you can try installing Unity to get it to work, but that is definitely not a good solution. I'll keep looking if I can find something.

**Comment by IonicaBizau** [comment]
@anmoljagetia No, it uses Openbox (not Gnome, nor Unity). I will add you as contributor in `package.json` :smile: You could try installing Ubuntu Server in a virtual machine.

**Comment by anmoljagetia** [comment]
In that case I'm pretty sure it won't work. It depends on Gnome to work, and since unity is based on gnome, it has the underlying packages, so it works on Unity too. I ported the script to work with mate machines, which looks something like : 

`gsettings set org.gnome.Terminal.Legacy.Profile:/org/gnome/terminal/legacy/profiles:/:$DEFAULT_PROFILE/ background-color $bg_color`

so to get it to work on Openbox, we will have to find the exact settings that OpenBox uses. And thanks a lot for the credit, :smile: , Although I already got good amount of points and credit for my work on AskUbuntu (thanks to you!). Thanks a lot! :smile:
